frontend/src/components/controls/index.js

```diff
@@ -19,11 +19,26 @@ const playButton = (cb, playbackState, disabled) => (
   </Button>
 )
 
+const pauseButton = (cb, playbackState, disabled) => (
+  <Button
+    onClick={cb}
+    animated='vertical'
+    disabled={(playbackState === MopidyApi.PAUSED || playbackState === MopidyApi.STOPPED || disabled)}
+    active={(playbackState === MopidyApi.PAUSED)}
+    className='jb-pause-button'
+  >
+    <Button.Content hidden>Pause</Button.Content>
+    <Button.Content visible>
+      <Icon name='pause' />
+    </Button.Content>
+  </Button>
+)
+
 const stopButton = (cb, playbackState, disabled) => (
   <Button
     onClick={cb}
     animated='vertical'
-    disabled={(playbackState === MopidyApi.STOPPED || disabled)}
+    disabled={(playbackState === MopidyApi.STOPPED || playbackState === MopidyApi.PAUSED || disabled)}
```

Comment:
Just wondering whether this is correct, for the "Stop" button to be disabled when `playbackState === MopidyApi.PAUSED`?

Should the user not be able to click "Stop" when the playback state is "Paused"?

Reply:
Fair point. We've never had the two ways to halt playback on screen at the same time. I think the answer is 'they should' so I will fix this.

Reply:
Many thanks :thumbsup:

Reply:
no worries. I added some extra specs to make sure I was actually testing this too